example/storybook/src/components/Disclosure/Accordion/index.stories.mdx

```diff
@@ -296,13 +296,13 @@ We provide in-depth information on how to customize and extend the component's f
 
 ### Customizing the Accordion
 
-We have a function called createAccordion which can be used to create a custom accordion component. This function takes in a configuration object which contains the styled components that you want to use for the Accordion. You can refer [style.gluestack.io](https://style.gluestack.io) for more information on how to use styled components.
+We have a function called createAccordion which can be used to create a custom accordion component. This function takes in a configuration object which contains the styled components that you want to use for the Accordion. You can refer [gluestack.io/style](/style) for more information on how to use styled components.
 
-You can read more on how to animate components [here](https://ui.gluestack.io/docs/advanced/animations).
+You can read more on how to animate components [here](/ui/docs/advanced/animations).
 
 ### Usage
 
-Default styling of all these components can be found in the `components/core/accordion` file. For reference, you can view the [source code](https://github.com/gluestack/gluestack-ui/tree/main/example/storybook/src/components/Disclosure/Accordion) of the styled `Accordion` components.
+Default styling of all these components can be found in the `components/core/accordion` file. For reference, you can view the [source code](https://github.com/gluestack/gluestack-ui/blob/main/packages/themed/src/components/Accordion) of the styled `Accordion` components.
```

Comment:
(https://github.com/gluestack/gluestack-ui/tree/main/example/storybook/src/components/Disclosure/Accordion)

correct it